### ISSUE TITLE
Constant beeping when playing 1st order ambisonic audio in Unity app deployed on Samsung S7

### ISSUE THREAD

**Issue by erikSik** [resonance-audio/resonance-audio-unity-sdk#21]
I'm developing an app using Unity (2017.2.03f) and Google Resonance (v1.1.1) for mobile phones. When testing the application on the Samsung S7, there is a loud constant beep heard (sounds like a sine wave at about 4270 Hz) in the scenes where there are 1st order ambisonc audio playing. This issue did not occur when testing the app on the two other devices that I have at hand - Pixel XL or the Nexus 5x.

Also, in some scenes, I've used a lowpass filter on the ResonanceAudioMixer, which seem to be able to reduce the level of the beep, but it is still too intrusive.

1. Have anyone had any similar issues with the Samsung S7, or any other phones?
2. Does anyone have any idea about what could be causing the issue?

The audio sample in question can be downloaded here: https://drive.google.com/open?id=1tFHO9hez6vOChDokZ0Lb9YcRtbLwZ04r 

Any input would be highly appreciated!

**Comment by jkammerl** [comment]
Thanks for the report. Could you check if this happens with other ambisonic render plugins (like OculusSpatializer) as well? Given that our plugin doesn't perform any platform specific rendering, it could be a Unity/S7 platform issue.

**Comment by erikSik** [comment]
Thanks! Yes, I'll give that a try.
I've just tested the app on a Samsung S8 and got the same issue with the beep, but slightly less loud on this model.

**Comment by erikSik** [comment]
I made a test also with the OculusSpatializer on the S7 and it also beeps. I have also made sure that all the sound enhancements (in the Samsung soundalive settings) were turned off. Have you had the chance to run any tests on any Samsung phones?

**Comment by erikSik** [comment]
Same beep happens on the S6 edge, but much quieter than the S7 and S8.

**Comment by erikSik** [comment]
I finally found the cause of this issue, and it was after all not related to the Resonace plugin, sorry for the false alarm!

It turned out that the recordings, done with the Sennheiser Ambeo mic contained unwanted high frequency content (not sure how it got there) at around 19.7 kHz, which in turn made three generations of Samsung phones and an iPhone to produce a beep at about 4.3 kHz. When the high frequency content was removed, the beep also dissapeared.
